windows/CodePush.cs

```diff
@@ -0,0 +1,668 @@
+﻿using System;
+using System.Collections.Generic;
+using ReactNative.Bridge;
+using ReactNative.Modules.Core;
+using ReactNative.UIManager;
+using Windows.ApplicationModel;
+using Windows.Storage;
+using System.IO;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+using System.Reflection;
+using Windows.Web.Http;
+using Windows.Storage.FileProperties;
+using ReactNative;
+
+namespace CodePush.ReactNative
+{
+    public class CodePushModule : IReactPackage
+    {
+        private static bool needToReportRollback = false;
+        private static bool isRunningBinaryVersion = false;
+        private static bool testConfigurationFlag = false;
+
+        private bool didUpdate = false;
```

Comment:
Like we've been doing in Objective-C, I believe the naming convention for instance variables is to camel casing (like you're doing), but to prefix the names with an underscore.